src/Microsoft.AspNet.Identity/PrincipalExtensions.cs

```diff
@@ -12,36 +12,6 @@ namespace System.Security.Claims
     public static class PrincipalExtensions
     {
         /// <summary>
-        /// Returns the Name claim value if present otherwise returns null.
-        /// </summary>
-        /// <param name="principal">The <see cref="ClaimsPrincipal"/> instance this method extends.</param>
-        /// <returns>The Name claim value, or null if the claim is not present.</returns>
-        /// <remarks>The Name claim is identified by <see cref="ClaimsIdentity.DefaultNameClaimType"/>.</remarks>
-        public static string GetUserName(this ClaimsPrincipal principal)
-        {
-            if (principal == null)
-            {
-                throw new ArgumentNullException(nameof(principal));
-            }
-            return principal.FindFirstValue(ClaimsIdentity.DefaultNameClaimType);
-        }
-
-        /// <summary>
-        /// Returns the User ID claim value if present otherwise returns null.
-        /// </summary>
-        /// <param name="principal">The <see cref="ClaimsPrincipal"/> instance this method extends.</param>
-        /// <returns>The User ID claim value, or null if the claim is not present.</returns>
-        /// <remarks>The User ID claim is identified by <see cref="ClaimTypes.NameIdentifier"/>.</remarks>
-        public static string GetUserId(this ClaimsPrincipal principal)
-        {
-            if (principal == null)
-            {
-                throw new ArgumentNullException(nameof(principal));
-            }
-            return principal.FindFirstValue(ClaimTypes.NameIdentifier);
-        }
-
-        /// <summary>
         /// Returns true if the principal has an identity with the application cookie identity
         /// </summary>
         /// <param name="principal">The <see cref="ClaimsPrincipal"/> instance this method extends.</param>
```

Comment:
While you are here, could you fix the null-conditional operator usage in the method bellow? E.g. principal is never null after the check. Or is this the method that is going away per #630?